### DISCUSSION TITLE
FR - Filesystem (Remote Share) - rename share folder

### DISCUSSION THREAD

**Discussion by patienttruth** [music-assistant/support#3209] [Q&A] [answered]
I've added my podcast directory as another remote share which is working great, however it's got the same name as my music directory.

Would it be possible to rename the share/folder in MA?

![Screenshot_20241120-203907](https://github.com/user-attachments/assets/17a7e8f2-fa94-4bc0-bcf7-1bf1a96fbf9f)

**Comment by OzGav** [comment, marked as answer]
You can do that in the MA settings

**Reply by patienttruth** [reply]
Doh! 🤦

Thank you!
